Polyhedron/demo/Polyhedron/opengl_tools.h

```diff
@@ -26,7 +26,6 @@
 #ifndef CGAL_OPENGL_TOOLS_H
 #define CGAL_OPENGL_TOOLS_H
 
-# include <CGAL/gl.h>
```

Comment:
You should remove the `Color` class from this file, because it might no longer compile without the declaration of OpenGL functions.